gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord, Angle
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
+    -----------
+    obs_table:  `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos: `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    @property
+    def offset(self):
+        """
+        Compute offsets of the different observations
+        
+        Returns
+        -------
+        vector : `~astropy.Angle`
+            Offsets
+        """
+        pnt_pos = SkyCoord(self.obs_table['RA_PNT'],
+                           self.obs_table['DEC_PNT'],
+                           unit='deg')
+
+        offset = pnt_pos.separation(self.target_pos)
+        
+        return offset.degree
+    
+    def plot_zenith_distribution(self, ax=None, bins=None):
+        """
+        Construct the zenith distribution of the observations
+        
+        Parameters
+        ----------
+        ax : `~matplolib.axes`, optional
+            Axis
+        bins : integer
+            number of bins, optional
+        range : `range`
+            range of the x axis, optional
+
+        Returns
+        --------
+        ax : `~matplolib.axes`
+            Axis
+        """
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+
+        zenith = self.obs_table['ZEN_PNT']
+
+        if bins is None:
+            bins = np.linspace(0, zenith.max() + 5, 30)
+
+        ax.hist(zenith, bins=bins)
+        ax.set_title('Zenith distribution')
+        ax.set_xlabel('Zenith (Deg)')
+        ax.set_ylabel('#Entries')
+        
+        return ax
+
+    def plot_offset_distribution(self, ax=None, bins=None):
+        """
+        Construct the offset distribution of the observations
+        
+        Parameters
+        ----------
+        ax : `~matplolib.axes`, optional
+            Axis
+        bins : integer
+            number of bins, optional
+
+        Returns
+        -------
+        ax : `~matplolib.axes`
+            Axis
+        """
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+        
+        offset = self.offset
+
+        if bins is None:
+            bins = np.linspace(0, offset.max()+0.5, 10)
+        ax.hist(offset, bins=bins)
+        ax.set_title('Offset distribution')
+        ax.set_xlabel('Offset (Deg)')
+        ax.set_ylabel('#Entries')
+        
+        return ax
+
+    def __str__(self):
```

Comment:
Please add a test for `__str__`.
It should look something like this:
https://github.com/gammapy/gammapy/blob/f10d310718c0a3c58e755e58094725dd276e91e4/gammapy/catalog/tests/test_hess.py#L57

The main point is to call `str(obs_summary)` somewhere to execute this.
Otherwise what will happen for such code that isn't covered is that someone changes the class and this method breaks.
It would be nice to have an assert, it could just be this:
```
text = str(obs_table)
assert 'Summary report' in text
```
although if you want to assert on more or other parts of the text, that's fine as well of course.